src/getOptions.js

```diff
@@ -34,6 +34,10 @@ function getOptions(loaderContext) {
     }
   }
 
+  if (typeof options.modifyVars === 'function') {
+    options.modifyVars = options.modifyVars(loaderContext);
```

Comment:
What's the idea behind passing the `loaderContext` to `modifyVars`? I'm trying to avoid these kind of extra features that make it hard to migrate/change anything later since we already exposed it to the outside.